.appveyor.yml

```diff
@@ -0,0 +1,25 @@
+# Test against the latest version of this Node.js version
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "8"
+    - nodejs_version: "9"
```

Comment:
Do we really want it to run on an non-lts version @nitriques ?

Reply:
Yes. When 10 goes out, we will switch to 10 and add 11 when possible.
We should always test against the latest version. I am open to let it fail.

Reply:
Perfect. If the tests fails with a non-lts version, will the badge turn in an error state?

Can (and should?) we ignore non-lts build error to keep the badge green?

Reply:
It is safe to ignore IHMO

Check https://www.appveyor.com/docs/build-configuration/#failing-strategy (matrix.allow_failure)

And  https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail  (matrix.allow_failure)